src/daemon/rpmostreed-os.c

```diff
@@ -1852,7 +1837,12 @@ rpmostreed_os_load_internals (RpmostreedOS *self, GError **error)
                                       booted_variant);
 
   if (!default_variant)
-    default_variant = rpmostreed_deployment_generate_blank_variant ();
+    {
+      if (booted_variant)
```

Comment:
We just set `booted_variant` right above. Seems like we can just drop this if-statement and always do `g_variant_ref (booted_variant)` ?